contracts/queue/tests/integration.rs

```diff
@@ -0,0 +1,94 @@
+use cosmwasm_std::testing::{mock_env, MockApi, MockStorage};
+use cosmwasm_std::{coin, from_slice, Env, HumanAddr};
+use cosmwasm_vm::testing::{handle, init, mock_instance, query};
+use cosmwasm_vm::Instance;
+
+use queue::contract::{CountResponse, HandleMsg, InitMsg, QueryMsg, State, SumResponse};
+
+/**
+This integration test tries to run and call the generated wasm.
+It depends on a release build being available already. You can create that with:
+
+cargo wasm && wasm-gc ./target/wasm32-unknown-unknown/release/hackatom.wasm
```

Comment:
I think this line can be simplified to just `cargo wasm`

Reply:
Yup. And with the RUST_FLAGS as well maybe....